scss/atoms/typography/_mixins.scss

```diff
@@ -0,0 +1,26 @@
+// ==========================================================================
```

Comment:
Expanded does not output single line comments. No style does.

Compressed, expanded, and nested support outputting multiline comments (without doing a forced comment `/*! */`).